build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
   id 'java-gradle-plugin'
   id "org.jetbrains.kotlin.jvm" version "1.3.61"
   id "com.gradle.plugin-publish" version "0.11.0"
+  id "org.gradle.kotlin.kotlin-dsl" version "1.1.0"
```

Comment:
I'd like to reduce the changes to the bare minimum @eskatos.
The plugin currently use Gradle is 5.6.4, do you happen to know which kotlin dsl version I should set in order to get rid of any warning and still have a working implementation?

Reply:
Gradle 5.6.4 works with version `1.2.9`

Reply:
Thanks @eskatos, I'll upgrade the kotlin-dsl plugin version to 1.2.9.
While doing that I'm still getting a warning:

> The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.3.41` that might work differently than in the requested version `1.3.61`.

I could probably get rid of it upgrading Kotlin, Gradle and kotlin-dsl plugin but once again I'd like to minimize the changeset and my impression is that it can be safely ignored. Please let me know if you think otherwise.